Fix `condition` and `fix` in submodels (#892)
TuringLang/DynamicPPL.jl: docs/src/api.md

## Fixing and unfixing

> We can also _fix_ a collection of variables in a [`Model`](@ref) to certain using [`DynamicPPL.fix`](@ref).

Reviewer: ```suggestion
We can also _fix_ a collection of variables in a [`Model`](@ref) to certain values using [`DynamicPPL.fix`](@ref).
```

> This might seem quite similar to the aforementioned [`condition`](@ref) and its siblings,

Reviewer: ```suggestion
This is quite similar to the aforementioned [`condition`](@ref) and its siblings,
```
